setup/index.js

```diff
@@ -31,13 +31,20 @@ execSync('rm -rf ios/HelloWorld/Base.lproj/LaunchScreen.xib', { cwd: rootDirecto
 execSync('yarn hygen setup splashscreen');
 
 console.log('\n🗑  Removing cruft...');
-deleteFile('../.babelrc'); // metro bundler version uses babel.config.js
 deleteFile('../.flowconfig');
 deleteFile('../App.js');
 deleteFile('../.gitattributes'); // not sure why this is here?
 execSync('rm -rf setup', { cwd: rootDirectory });
 execSync('rm -rf .git', { cwd: rootDirectory }); // blow away old repo if there
 
+console.log('\n📱 Setting initial version @0.0.1 ...');
+execSync(
+  'yarn add -D react-native-version && ./node_modules/.bin/react-native-version  --never-increment-build && yarn remove react-native-version',
```

Comment:
could this / should this leverage `npx` instead of adding and removing?

Reply:
Totally possible, I can look into that @cball, honestly have no idea how that all works I haven't really dove into using `npx` at this point.

Reply:
Updated to use NPX.